### DISCUSSION TITLE
How to get Trail Visibility quest?

### DISCUSSION THREAD

**Discussion by ERYpTION** [Helium314/SCEE#556] [Q&A] [answered]
I have enabled the 'What is the Trail Visibility of this way?' quest but then don't see these quests for ways on the map where I expect. Should a highway=path + surface=ground trigger the Trail visibility quest?

**Comment by mcliquid** [comment, marked as answer]
Only if the path has also a "sac_scale" mapped which is not "hiking" but above.

**Comment by Helium314** [comment]
In general, you can check how to get (most) quests by clicking the cog icon in quest selection.
For quests that use simple filtering by tags, this opens a dialog in which you can see and edit the filter (works only in expert mode for obvious reasons).

**Reply by ravenfeld** [reply]
Thank you very much for adding this because in my region I had to be simpler than the request you put in
